SPLASH/res/shaders/compute/ParticleComputeShader.hlsl

```diff
@@ -53,32 +54,32 @@ void createParticle(float3 v0, float3 v1, float3 v2, float3 v3, int particleInde
 	CSOutputBuffer[v0Index].bitangent = 0.f;
 	
 	CSOutputBuffer[v0Index + 1].position = v1;
```

Comment:
Do something like below to reduce unnecesary map lookups. For all indices
auto& buffer1 = CSOutputBuffer[v0Index + 1]